Gems/Atom/Feature/Common/Assets/Passes/LowEndPipelineXRLeft.pass

```diff
@@ -0,0 +1,373 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "PassAsset",
+    "ClassData": {
+        "PassTemplate": {
+            "Name": "LowEndPipelineXRLeftTemplate",
```

Comment:
Can you explain why you have two pipeline passes for left and right which are exactly same with only different template names? 

I think you can just use one template create two different render pipelines.

Reply:
Sounds good.